src/window/mod.rs

```diff
@@ -309,6 +311,8 @@ pub fn main_loop(
     let cmd_line_settings = SETTINGS.get::<CmdLineSettings>();
     let mut window_wrapper = WinitWindowWrapper::new(window, initial_window_size);
 
+    crate::clipboard::init(&event_loop);
```

Comment:
Could this be moved before calling `main_loop` so that we don't need to duplicate the call on Windows and the other platforms?

Reply:
Fixed!